packages/api/src/generated/schema.graphql

```diff
@@ -803,6 +803,7 @@ type Query {
   me: User
   newsletterEmails: NewsletterEmailsResult!
   reminder(linkId: ID!): ReminderResult!
+  search(after: String, first: Int, query: String): SearchResult!
```

Comment:
@satindar this new method will replace `GetArticles` for us. Returns a kind of slimmed down object, allows it to be either a deep link to a highlight or a page.